nixos/modules/services/mail/dovecot.nix

```diff
@@ -429,7 +429,9 @@ in
 
       startLimitIntervalSec = 60;  # 1 min
       serviceConfig = {
+        Type = "notify";
         ExecStart = "${dovecotPkg}/sbin/dovecot -F";
+        ExecStop = "${dovecotPkg}/sbin/doveadm stop";
```

Comment:
I'm not sure if this actually does what we want. Looking at the [relevant systemd documentation](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStop=), one thing sticks out: "it is usually not sufficient to specify a command for this setting that only asks the service to terminate (for example, by sending some form of termination signal to it), but does not wait for it to do so". Does doveadm do that?